packages/graphql_server/src/graphql/resolvers.ts

```diff
@@ -1,8 +1,9 @@
-import { insertProject, parseGitHubUrl } from '../processRepo'
+import { insertProject, parseGitHubUrl, exposeProjectsData } from '../processRepo'
 
 const resolvers = {
   Query: {
-    helloWorld: () => 'Hello world!'
+    helloWorld: () => 'Hello world!',
+    allProjects: () => exposeProjectsData()
```

Comment:
When starting the server via `npm run dev` it crashes. That is because you have not modified the `/graphql/schema.ts` file accordingly.